src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adler(num) {
```

Comment:
The adder function should accept multiple arguments, consider using a rest parameter (...args).